### DISCUSSION TITLE
Missing documentation

### DISCUSSION THREAD

**Discussion by josephgarnier** [efemkay/obsidian-modular-css-layout#88] [Q&A] [answered]
First of all, thank you for this very useful and helpful project.

In the [css `MCL Multi Column.css`](https://github.com/efemkay/obsidian-modular-css-layout/blob/main/MCL%20Multi%20Column.css), I see that it's possible to use the metadata `fw-x`, `pw-x`, `mw-x`. But **they are not documented**. Is there a particular reason? Are they depreciated?

**Comment by efemkay** [comment, marked as answer]
thanks for actually going through the code and find out those code chunks for finer control. yeah, it was not documented in the official document site as i was experimenting with new document site. and then i got busy with other things.

but since you are interested, here's the direct page to the new docs

https://obsidian-mcl-docs.netlify.app/06-advanced/callout-width-control/
